source/WorkingPractices/Diagnostics/lfric_diagnostics.rst

```diff
@@ -1,11 +1,409 @@
++-----------------------------------+----------------------------------------+
+|  :math:`\mathbb{W}_{2v}`          |  ``grid_ref="full_level_face_grid"``   |
++-----------------------------------+----------------------------------------+
+|  :math:`\mathbb{W}_{2h}`          |  ``grid_ref="half_level_edge_grid"``   |
++-----------------------------------+----------------------------------------+
+|  :math:`\mathbb{W}_{theta}`       |  ``grid_ref="full_level_face_grid"``   |
++-----------------------------------+----------------------------------------+
+|  :math:`\mathbb{W}_{theta}`       |  ``grid_ref="full_level_face_grid"``   |
++-----------------------------------+----------------------------------------+
+|  Single-level field               |  ``domain_ref="face"``                 |
++-----------------------------------+----------------------------------------+
+|  Multi-data field                 |  ``domain_ref="face"``                 |
+|                                   |  ``axis_ref="<multidata type>"``       |
++-----------------------------------+----------------------------------------+
+
+A multi-data field is often called a tiled field, and contains more
+than one related quantity. For multi-data fields, the ``<multidata
+type>`` text would be replaced by one of the multidata field types
+used in the model and defined in the ``axis_def_main.xml`` file in the
+``lfric_atm/metadata`` directory. For example the following field is
+on surface tiles.
+
+.. code-block:: xml
+
+    <field id="surface__throughfall" name="throughfall_rate"
+    long_name="canopy_throughfall_flux" unit="kg m-2 s-1"
+    domain_ref="face" axis_ref="surface_tiles" />
+
+The number of quantities in each type of multi-data field is defined
+within the application.
+
+Initialising the field
+~~~~~~~~~~~~~~~~~~~~~~
+
+Initialising the field relates to defining the LFRic function space
+that the field lives on rather than initialising the values held in
+the field.
+
+If a field is not available to hold the diagnostic data, then one must
+be declared local to the diagnostic routine and initialised to be the
+correct field type.
+
+The following code is from an existing science scheme algorithm.  It
+declares some fields and passes them to the scheme's diagnostic
+initialisation procedure.
+
+.. code-block:: fortran
+
+    type( field_type ) :: soil_moisture_content
+    type( field_type ) :: grid_canopy_water
+    type( field_type ) :: throughfall
+    type( field_type ) :: grid_throughfall
+
+    call initialise_diags_for_jules_soil(soil_moisture_content,  &
+                                         grid_canopy_water,      &
+                                         throughfall,            &
+                                         grid_throughfall)
+
+The ``initialise_diags_for_jules_soil`` procedure calls the LFRic
+atmosphere ``init_diag`` function:
+
+.. code-block:: fortran
+
+    soil_moisture_content_flag = init_diag(soil_moisture_content, &
```

Comment:
Being picky, it would be better to align the continuation lines

Reply:
Changed all the alignment of these lines.

Reply:
Cheers